engine/src/main/java/org/terasology/logic/inventory/ItemComponent.java

```diff
@@ -85,4 +79,5 @@
 
     public Prefab pickupPrefab;
 
+    public int cooldownTime = 200;
```

Comment:
This prevents prefabs that do not specify a cooldown to get a better value than "machine gun"